adafruit_tcs34725.py

```diff
@@ -314,3 +231,88 @@ def max_value(self):
     @max_value.setter
     def max_value(self, val):
         self._write_u16(_REGISTER_AIHT, val)
+
+    def _temperature_and_lux_dn40(self):
+        """Converts the raw R/G/B values to color temperature in degrees
+        Kelvin using the algorithm described in DN40 from Taos (now AMS).
+        Also computes lux. Returns tuple with both values or tuple of Nones
+        if computation can not be done.
+        """
+        # pylint: disable=bad-whitespace, invalid-name, too-many-locals
+
+        # Initial input values
+        ATIME = self._read_u8(_REGISTER_ATIME)
+        ATIME_ms = (256 - ATIME) * 2.4
+        AGAINx = self.gain
+        R, G, B, C = self.color_raw
+
+        # Device specific values (DN40 Table 1 in Appendix I)
+        GA = 1.0            # Glass Attenuation Factor
```

Comment:
I wonder if we should expose this as a parameter outside of this function? This value would be adjusted to take into account an enclosure, for example, such as when the TCS34725 is placed behind a glass or plastic screen which will cause some attentuation of the signal? I didn't to this on the Arduino driver, but seeing this code here now it seems like a good idea to break this out, keeping the current value of `1.0` as a sensible default which assumes nothing between the light source and the photodiodes on the device.

Reply:
It'd be trivial to add. Create a property and use it here. What should it be called? `glass_attenuation`? `glass_attenuation_factor`? `GA`? other?

Reply:
`attenuation` seems sensible and short enough? That's the basic idea anyway, and it extends beyond just 'glass'.

Reply:
Hmm .... `attenuation_factor` is actually clearer maybe, but seems kind of longish??? I don't think I'd necessarily use that in C, but I'm not intimately familiar with Python naming conventions and norms at this point either.

Reply:
I kind of feel like "glass" should be in the name. From what I've seen, Python community prefers long explicit names. Naming things is hard...maybe others will chime in...

Reply:
I'm OK with `glass_attenuation` then, for my +1.  It's at least clear and matches the technical documentation.